drtprod: set ROACHPROD_GCE_DEFAULT_SERVICE_ACCOUNT to match ROACHPROD_GCE_DEFAULT_PROJECT

srosenberg opened the pull request:
Minor tweak to ensure the correct default
service account is set.

Epic: none
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/153134)
<!-- Reviewable:end -->

srosenberg committed:
drtprod: set ROACHPROD_GCE_DEFAULT_SERVICE_ACCOUNT to match ROACHPROD_GCE_DEFAULT_PROJECT

Minor tweak to ensure the correct default
service account is set.

Epic: none
Release note: None

srosenberg commented:
TFTR!

bors r=shailendra-patel,herkolategan
